site/assets/js/map.js

```diff
@@ -1,3 +1,5 @@
+var g, svg;
```

Comment:
looks like we don't need to hoist `svg` at all (isn't used outside of this file afaict)